tests/Microsoft.Identity.Test.Performance/Microsoft.Identity.Test.Performance.csproj

```diff
@@ -2,12 +2,15 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
good catch

Reply:
It shouldn't matter if there is one target in TargetFrameworks element, but I think it wouldn't build for me sometimes when I had it like that. Either way this is work in progress. Might just change to use net6. (Although netcore3.1 is slower in general than net6, so maybe beneficial to have lower bound for perf tests too).